tests/arrows/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -304,10 +304,11 @@ export const bem = block =>
 <FlatList
   renderItem={(
     info // $FlowExpectedError - bad widgetCount type 6, should be Object
-  ) =>
+  ) => (
     <span>
       {info.item.widget.missingProp}
-    </span>}
+    </span>
+  )}
```

Comment:
This is great, people complained about the fact that `}` was on the same line as `}` as well.